google-cloud-storage/src/main/java/com/google/cloud/storage/PackagePrivateMethodWorkarounds.java

```diff
@@ -86,17 +86,6 @@ public static Function<WriteChannel, Optional<BlobInfo>> maybeGetBlobInfoFunctio
     };
   }
 
-  public static Function<WriteChannel, Optional<BlobInfo>> maybeGetBlobInfoFunction() {
```

Comment:
In `main` this method is still used in two locations. How is it's removal here not causing a failure?

Reply:
its a duplicate that was not removed during the merge up from main, we don't need two implementations of the same method.